### PR TITLE
bytefield:0.0.6

### PR DESCRIPTION
**Pull request by jomaway**
I am submitting
- [x] an update for a package

Description: Removed tablex dependencies and some minor improvements

### CODE REVIEW

**Comment by elegaanz** [comment]
There are a few places where you use snake_case for public functions or their arguments, while you seem to use kebab-case in most place. Is that intended or not? And if not, do you want me to list all the occurrences of snake_case I found?

**Comment by jomaway** [comment]
I tried to replace all snake_case functions. But maybe I forgot some.

If you could highlight those you found would be very nice 👍.

I will then replace them later this evening.

File: packages/preview/bytefield/0.0.6/lib/api.typ [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+  note(side, level:level, rowspan:rowspan, bracket:bracket, content)
+}
+
+/// Shows a special note with a start_addr (top aligned) and end_addr (bottom aligned) on the left of the associated row.
+///
+/// #emoji.warning *experimental:* This will probably change in a future version.
+///
+/// - start_addr (string, content):  The start address will be top aligned
+/// - end_addr (string, content): The end address will be bottom aligned
+#let section(start_addr, end_addr, span: 1) = {
```
Comment (elegaanz) [inline packages/preview/bytefield/0.0.6/lib/api.typ:166]: The two first arguments here.

File: packages/preview/bytefield/0.0.6/lib/states.typ [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+#let default_header_border = state("bf-header-border", none);
+
+// function to use with show rule
+#let bf-config(
+  row_height: 2.5em,
+  field_font_size: auto,
+  note_font_size: auto,
+  header_font_size: 9pt,
+  header_background: none,
+  header_border: none,
```
Comment (elegaanz) [inline packages/preview/bytefield/0.0.6/lib/states.typ:16]: All these arguments.

**Review by elegaanz — COMMENTED** [review]
There are other places where snake_case is still used, but it is not in the public interface, so it is not as important I think. Thank you for taking time to fix this. :)

**Comment by jomaway** [comment]
Oh true I just refactored function names. Didn't think about parameters 🙈

File: packages/preview/bytefield/0.0.6/lib/states.typ (hunk 2 above)
Comment (jomaway) [reply packages/preview/bytefield/0.0.6/lib/states.typ]: fixed

**Comment by jomaway** [comment]
OK, i fixed the one you mentioned and some more i found. So you can merge.

**Comment by elegaanz** [comment]
Perfect, thank you :)

**Pull request merged**